README.md

```diff
@@ -48,14 +48,17 @@ Then, you can use the `FlaskPooledClusterRpcProxy` singleton just as you would n
 
 `FlaskPooledClusterRpcProxy` accepts all nameko configuration values, prefixed with the `NAMEKO_` prefix. In addition, it exposes additional configuration options:
 
-* `INITIAL_CONNECTIONS (int, default=2)` - the number of initial connections to the Nameko cluster to create
-* `MAX_CONNECTIONS (int, default=8)` - the max number of connections to the Nameko cluster to create before raises an error
+* `NAMEKO_INITIAL_CONNECTIONS (int, default=2)` - the number of initial connections to the Nameko cluster to create
+* `NAMEKO_MAX_CONNECTIONS (int, default=8)` - the max number of connections to the Nameko cluster to create before raises an error
+* `NAMEKO_LAZY_LOAD_SERVICCES (bool, default=False)` - whether connections to services should be loaded when the service is accessed (False) or when a method is called on a service (True)
```

Comment:
I think the naming here is a little confusing -- that's because I would imagine "lazy load services", if true, to only create a connection when I access a service -- which is the behavior when it is false. 

what do you think about NAMEKO_CONNECT_ON_METHOD_CALL?